### DISCUSSION TITLE
Getting error on installing the dependencies

### DISCUSSION THREAD

**Discussion by sudhanshuku01** [RocketChat/Rocket.Chat.ReactNative#5519] [General]
shows this error after hitting the $ npm install

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: rocket-chat-reactnative@4.47.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0" from @react-native-community/picker@1.8.1
npm ERR! node_modules/@react-native-community/picker
npm ERR!   @react-native-community/picker@"^1.8.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! C:\Users\sudhanshu kumar\AppData\Local\npm-cache\_logs\2024-01-28T09_46_30_467Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: C:\Users\sudhanshu kumar\AppData\Local\npm-cache\_logs\2024-01-28T09_46_30_467Z-debug-0.log

**Comment by dnlsilva** [comment]
Follow the docs and use `yarn`: https://developer.rocket.chat/open-source-projects/mobile-app#set-up-the-codebase
